distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -2927,3 +2928,7 @@ apply_unary_map column:Column new_name:Text function expected_result_type:Value_
     Java_Problems.with_map_operation_problem_aggregator column.name Problem_Behavior.Report_Warning java_problem_aggregator->
         map_column = UnaryOperation.mapFunction column.java_column function nothing_unchanged storage_type new_name java_problem_aggregator
         Column.Value map_column
+
+## PRIVATE
+Many_Files_List.from (that : Column) =
+    Many_Files_List.Value that that.to_vector
```

Comment:
Throw if an invalid value type (only allow Text of Mixed (File))?

Reply:
Done